plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts

```diff
@@ -390,29 +390,48 @@ export class GraphQLInstrumentation extends InstrumentationBase {
     });
   }
 
+  private getOperationType(
+    operation: graphqlTypes.DefinitionNode | undefined
+  ): string | undefined {
+    if (operation && operation.kind == 'OperationDefinition') {
+      return operation.operation;
+    }
+
+    return undefined;
+  }
+
+  private getOperationName(
+    operation: graphqlTypes.DefinitionNode | undefined,
+    processedArgs: graphqlTypes.ExecutionArgs
+  ): string | undefined {
+    if (operation && operation.kind == 'OperationDefinition') {
+      if (operation.name) {
+        return operation.name.value;
+      }
+    }
+    if (processedArgs.operationName) {
```

Comment:
I dont think you need that if you return undefined anyway so if operationName is undefined it will return undefined anyway
```suggestion
    return processedArgs.operationName;
```

Reply:
Had to use `?? undefined` to make sure to return undefined, otherwise typescript complain about returning null.